index.js

```diff
@@ -78,7 +78,7 @@ const destroyCircular = ({
 		}
 
 		// TODO: Use `stream.isReadable()` when targeting Node.js 18.
-		if (typeof value === 'object' && typeof value.pipe === 'function') {
+		if (typeof value === 'object' && value && typeof value.pipe === 'function') {
```

Comment:
```suggestion
		if (value !== null && typeof value === 'object' && typeof value.pipe === 'function') {
```

Reply:
@rj-david Could you add suggested change? Because it's quite straightforward fix otherwise :-) Thank you 🙏